spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/data/TestSparkOrcReader.java

```diff
@@ -62,8 +62,7 @@ public void writeAndValidateRepeatingRecords() throws IOException {
 
   private void writeAndValidateRecords(Schema schema, Iterable<InternalRow> expected)
       throws IOException {
-    final File testFile = temp.newFile();
-    Assert.assertTrue("Delete should succeed", testFile.delete());
+    final File testFile = temp.resolve("test").toFile();
```

Comment:
This test suite also hit issues with JUnit 4 so I ported all the `AvroDataTest` suites to JUnit 5.